ui/src/app/dashboard/total-summary/total-summary.component.ts

```diff
@@ -9,5 +12,14 @@ import {StatusCounts} from "../../shared/model/StatusCounts";
 export class TotalSummaryComponent {
   @Input() summary: StatusCounts;
 
-  constructor() {}
+  constructor(private readonly activatedRoute: ActivatedRoute) {}
+
+  getUrlParams(statusCount: StatusCount) {
+    let map = new Map<string, string[]>();
+    map.set('projectId', [this.activatedRoute.snapshot.queryParams['projectId']]);
```

Comment:
I made it an array to match the specified type of the map (`Map<string, string[]>`), which is required for the `encodeURLSearchParamsFromMap()` method. Is there a way to avoid doing this?

Reply:
Ah no you're correct, sorry!